BothamUI/BothamLoadingUI.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  BothamLoadingUI.swift
+//  BothamUI
+//
+//  Created by Davide Mendolia on 24/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+
+public protocol BothamLoadingUI: BothamUI {
+    func showLoader()
+    func hideLoader()
```

Comment:
Don't you need to declare these methods as public like you are already doing with the protocol?